ee/codegen/src/__test__/nodes/__snapshots__/generic-node.test.ts.snap

```diff
@@ -19,6 +19,7 @@ exports[`GenericNode > basic > getNodeFile 1`] = `
 
 
 class MyCustomNode(BaseNode):
-    pass
+    class NodeTrigger(BaseNode.Trigger):
+        merge_behavior = "AWAIT_ALL"
```

Comment:
[oos] `MergeBehavior` is currently an enum: https://github.com/vellum-ai/vellum-python-sdks/blob/main/src/vellum/workflows/nodes/bases/base.py#L241

We likely should update it on the SDK side to be a literal of three string options